Document JSON-RPC API (#909)
jamulussoftware/jamulussoftware.github.io: wiki/en/Software-Manual.md

{% include_relative Include-Shared-Commands.md %}

# Controlling the Client via API

> In addition to the CLI, Jamulus can be controlled using an API. This is beneficial for advanced usecases - especially if there is no GUI or another application needs to interact with Jamulus. Please note that the API is still experimental. Information on the [JSON-RPC API can be found in the main repository](https://github.com/jamulussoftware/jamulus/blob/main/docs/JSON-RPC.md).

Reviewer: > usecases

"use cases".

> especially if there is no GUI or another application needs to interact with Jamulus

I'm not sure that adds anything.

Author: It's a valid use case and was meant as an example for clarification.

Reviewer: So, "for example" would be better?
```suggestion
In addition to the CLI, Jamulus can be controlled using an API. This is beneficial for advanced use cases - for example, where there is no GUI, or another application needs to interact with Jamulus. Please note that the API is still experimental. Information on the [JSON-RPC API can be found in the main repository](https://github.com/jamulussoftware/jamulus/blob/main/docs/JSON-RPC.md).
```

Author: Yes. Agree.
